File: app-mvp/feature-shot-details/src/main/java/com/bubbble/shotdetails/comments/NoCommentsViewHolder.java
package com.bubbble.shotdetails.comments;

import android.view.LayoutInflater;
import android.view.ViewGroup;

import androidx.recyclerview.widget.RecyclerView;

import com.bubbble.shotdetails.R;

public class NoCommentsViewHolder extends RecyclerView.ViewHolder {

    public NoCommentsViewHolder(LayoutInflater layoutInflater, ViewGroup parent) {
        super(layoutInflater.inflate(R.layout.item_no_comments, parent, false));
    }

}

File: app-mvp/feature-shot-details/src/main/java/com/bubbble/shotdetails/comments/NoCommentsViewHolder.kt
package com.bubbble.shotdetails.comments

import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.bubbble.shotdetails.R

internal class NoCommentsViewHolder(
    layoutInflater: LayoutInflater,
    parent: ViewGroup
) : RecyclerView.ViewHolder(layoutInflater.inflate(R.layout.item_no_comments, parent, false))